projectq/setups/decompositions/amplitudeamplification.py

```diff
@@ -0,0 +1,112 @@
+#   Copyright 2019 ProjectQ-Framework (www.projectq.ch)
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""
+Registers a decomposition for quantum amplitude amplification.
+
+(Quick reference https://en.wikipedia.org/wiki/Amplitude_amplification.
+Complete reference G. Brassard, P. Hoyer, M. Mosca, A. Tapp (2000)
+Quantum Amplitude Amplification and Estimation
+https://arxiv.org/abs/quant-ph/0005055)
+
+Quantum Amplitude Amplification (QAA) executes the algorithm, but not
+the final measurement required to obtain the marked state(s) with high
+probability. The starting state on wich the QAA algorithm is executed
+is the one resulting of aplying the Algorithm on the |0> state.
+
+Example:
+    .. code-block:: python
+
+       def func_algorithm(system_qubits):
```

Comment:
To be correct, the algorithm function in this example code should include the `eng` parameter (cf. code below)

Reply:
You are right. Thanks, Corrected